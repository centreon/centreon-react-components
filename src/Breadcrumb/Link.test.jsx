/* eslint-disable no-undef */

import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import BreadcrumbLink from './Link';

describe('Breadcrumb', () => {
  it('renders', () => {
    const breadcrumb = {
      label: 'first level',
      link: '/firstLevel',
    };

    const { container } = render(
      <MemoryRouter>
        <BreadcrumbLink index={0} count={1} breadcrumb={breadcrumb} />
      </MemoryRouter>,
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
