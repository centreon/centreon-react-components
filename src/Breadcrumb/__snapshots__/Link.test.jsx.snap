// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumb renders 1`] = `
.emotion-0 {
  font-size: 12px;
}

<a
  class="MuiTypography-root MuiLink-root MuiLink-underlineHover emotion-0 MuiTypography-colorTextPrimary"
  href="/firstLevel"
>
  first level
</a>
`;
